float64[] tcp_position
---
float64[] joint_position
bool success
